tests/ballerina-unit-test/src/test/resources/test-src/expressions/stamp/anydata-stamp-expr-test.bal

```diff
@@ -0,0 +1,93 @@
+type Employee record {
```

Comment:
Shall we add the license header to all the newly added ballerina test files?

Reply:
Added.